docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<blockquote>Small values in the Matrix Profile are called <em>motifs</em>, and large values are called <em>discords</em>.</blockquote><p> </p><p>I can see that this sentence is from the introduction part of the paper; however, I think it is not meaningful to call small value in matrix profile, a motif. Because a<strong> motif is a subsequence</strong> that has small value to its nearest neighbour.</p><p> </p><p>I think the authors provided a better definition in the abstract of the paper:</p><p>"<strong>Time series motifs refer to two particularly close subsequences, whereas time series discords indicate subsequences that are far from their nearest neighbors.</strong>"</p><p> </p><p> </p><blockquote>It may be usefull to score subsequences with</blockquote><p>I think the paper is clearer here. In the paper, it says: "....score subsequences with a <strong>meta-data that reflects that ...</strong>"</p><p> </p><p>Also, I think it would be a good idea to introduce <code>T(+)</code> and <code>T(-)</code> to the reader.</p><p> </p><blockquote>This is exactly the property we desire.</blockquote><p> </p><p>Why? According to my understanding and what I read in the paper, this property can be used to find a subsequence that can uniquely identify a class. In other words, this can be used in classification. Is that correct? I think it would be nice to provide an example to explain the significance of contrast profile. I know there is an example in paper (see Fig. 1), but I think it is a little bit complicated. So, let's try and see if we can come up with a good example to show the importance of contrast profile. Think about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>